docs/workflow/building/libraries/README.md

```diff
@@ -147,7 +147,7 @@ Similar to building the entire repo with `build.cmd` or `build.sh` in the root y
  ./build.sh -subsetCategory libraries src/libraries/System.Collections
 ```
 
-- Build just the tests for a library project.
+- Build just the tests for a library project
```

Comment:
```suggestion
- Build just the tests for a library project:
```

Reply:
Yes, we want to be consistent - hence no semicolon as no other example on the page has it.
I don't mind adding them everywhere, it is just not something we adopted on this page yet.